dataservice/util/data_gen/data.py

```diff
@@ -0,0 +1,121 @@
+"""
+This file contains numbers and choices for
+the entities
+"""
+import random
+import csv
+
+NUM_PARTICIPANTS = 10
+
+# NUM_DEMOGRAPHICS = 1
+
+NUM_DIAGNOSES = random.randint(0, 35)
+
+NUM_SAMPLES = random.randint(0, 65)
+
+NUM_ALIQUOTS = random.randint(0, 65)
+
+NUM_SEQ_EXP = 1
+
+NUM_GENOMIC_FILES = 5
+
+# Demographics data
+race_list = [
```

Comment:
Might be nice to put these in a `txt` file to stay consistent and have less clutter here.